src/User32.Desktop/User32.cs

```diff
@@ -107,7 +107,7 @@ public static partial class User32
         public static extern IntPtr GetForegroundWindow();
 
         [DllImport(nameof(User32))]
-        public static extern int SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
+        public static extern IntPtr SendMessage(IntPtr hWnd, int wMsg, IntPtr wParam, IntPtr lParam);
```

Comment:
This change causes hesitation because it's an API breaking change, yet it's not clear (to me) that it significantly better. `int` and `IntPtr` can already be cast between each other. and `LRESULT` isn't necessarily a pointer type, is it?